manifests/Microsoft/VisualStudioCode/1.46.1.yaml

```diff
@@ -0,0 +1,14 @@
+Id: Microsoft.VisualStudioCode
+Version: 1.46.1
+Name: Ladislau Szomoru
```

Comment:
This is the name of the application https://docs.microsoft.com/en-us/windows/package-manager/package/manifest?tabs=minschema%2Ccompschema